### PR TITLE
fix(deps): update module golang.org/x/crypto to v0.31.0

### PR DESCRIPTION
**Pull request by sairaj18**
Updated module golang.org/x/crypto to v0.31.0 to fix this [vulnerability](https://github.com/newrelic/nri-mssql/actions/runs/12351250398/job/34465817799?pr=190#:~:text=org/x/crypto%20%E2%94%82-,CVE%2D2024%2D45337,-%E2%94%82%20CRITICAL%20%E2%94%82%20fixed%20%20%E2%94%82%20v0.24.0)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,9 @@ Unreleased section should follow [Release Toolkit](https://github.com/newrelic/r
 
 ## Unreleased
 
+### security
+- fixed a security vulnerability by updating golang.org/x/crypto to v0.31.0, golang.org/x/text to v0.21.0
+
```
Comment (rajrohanyadav) [inline CHANGELOG.md:14]: Shouldn't it be just a `chore(deps)`? I think you can just mention it as a dependency update. IMO, Security updates are when you fix something in code of this repo that could have been exploited.

**Review by rajrohanyadav — CHANGES_REQUESTED** [review]
I believe it should be a dependency update rather than a security fix in `CHANGELOG.md`.

File: CHANGELOG.md (hunk 1 above)
Comment (sairaj18) [reply CHANGELOG.md:14]: when i use `### dependencies` the `changelog.md` validation is failing. I should be using anyone from the below

- **breaking:** A breaking change on a user-facing API. Rendered first and with a warning sign to quickly catch the attention of the reader.
- **security:** Security fixes that remediate potential or existing security vulnerabilities.
- **enhancement:** New features or improvements to existing ones.
- **bugfix:** Fixes to incorrect behavior of the application.

Comment (rajrohanyadav) [reply CHANGELOG.md:14]: Did you try `### dependency`?

Comment (sairaj18) [reply CHANGELOG.md]: made the change in this [commit](https://github.com/newrelic/nri-mssql/pull/191/commits/f414e38b9dcd04d60b6215a7524423f878f80712)

**Review by rajrohanyadav — APPROVED** [review]
LGTM

**Pull request merged**